codes/scala/chapter_computational_complexity/iteration.scala

```diff
@@ -0,0 +1,81 @@
+
+/**
+ * File: iteration.scala
```

Comment:
The format of the file header:

/**
 * File: iteration.java
 * Created Time: 2024-08-09
 * Author: WangBlue (wangjiping596@gmail.com)
 */